ui/components/canvas2d/src/Canvas2D.svelte

```diff
@@ -204,7 +204,11 @@
     for (const view of Object.values(selectedItem.views)) {
       zoomFactor[view.id] = 1;
       images[view.id] = new Image();
-      images[view.id].src = `/${view.uri}` || view.url;
+      if (utils.isValidURL(view.uri)) {
+        images[view.id].src = `${view.uri}` || view.url;
```

Comment:
I think `|| view.url` was added here during data migration to avoid bugs but should probably not be needed in the future?

Reply:
agreed, but in a future PR